src/io/github/sspanak/tt9/ime/KeyPadHandler.java

```diff
@@ -93,17 +93,6 @@ public void onFinishInputView(boolean finishingInput) {
 		onFinishTyping();
 	}
 
-	/**
```

Comment:
Please, don't do this. Applications are not very consistent in calling `onFinishInput()` and `onFinishInputView()`. They call the functions both when going to background mode and when the user switches to another input field in the same app. The thing is, in each scenario, applications could call either `onFinishInput()`, or `onFinishInputView()`, or both, despite the fact the documentation clearly states what they are for and when to call them.

This is the most stable solution I've found. It has been working for months without complaints, so let's not change anything.